tmt/base.py

```diff
@@ -311,8 +363,16 @@ def from_spec(cls, raw: _RawRequireFmfId) -> 'RequireFmfId':  # type: ignore[ove
             raise tmt.utils.SpecificationError(
                 f"The 'ref' field must be a string, got '{type(ref).__name__}'.")
 
-        return RequireFmfId(
-            **{key: cast(Optional[str], raw.get(key, None)) for key in cls.VALID_KEYS})
+        fmf_id = RequireFmfId()
+
+        for key in ('url', 'ref', 'name', 'destination', 'nick'):
```

Comment:
The `destination` key should be removed as it's covered below.

Reply:
Right, fixed in 87870ccc